tests/expected_report.txt

```diff
@@ -28,9 +28,9 @@ tests/input/traits-uses.php                           11      0
 tests/input/UnusedVariables.php                       1       0
 tests/input/useless-semicolon.php                     2       0
 ----------------------------------------------------------------------
-A TOTAL OF 215 ERRORS AND 0 WARNINGS WERE FOUND IN 24 FILES
+A TOTAL OF 217 ERRORS AND 0 WARNINGS WERE FOUND IN 24 FILES
```

Comment:
Am I wrong or shouldn't we only have 1 more error here from the tests? Is this because of conflicting with the ` !` rule?

Reply:
It's because of the conflict